### PR TITLE
Add VR Mode Setting

### PR DESCRIPTION
**Pull request by rcbevans**
Add VR Mode Setting which toggles the MainWindow background opacity between 0 and 255 for use with CrewChief VR Mirroring

### CODE REVIEW

File: Startup.cs [hunk 1]
```diff
@@ -33,6 +34,9 @@ public void ConfigureServices(IServiceCollection services)
     // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
     public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
     {
+        //if (env.IsDevelopment())
```
Comment (Yuvix25) [inline Startup.cs:37]: Is this something that's good to have in case I want to uncomment it or just a left-over?

File: Startup.cs [hunk 2]
```diff
@@ -182,6 +189,9 @@ private async Task CreateMainWindow()
             return;
         }
 
+        //if (env.IsDevelopment())
+        //    MainWindow.WebContents.OpenDevTools();
```
Comment (Yuvix25) [inline Startup.cs:193]: Not needed - please remove.

File: Startup.cs [hunk 3]
```diff
@@ -397,6 +407,8 @@ private async Task CreateSettingsWindow(IWebHostEnvironment env)
 
         if (!env.IsDevelopment())
             SettingsWindow.RemoveMenu();
+        //else
+        //    SettingsWindow.WebContents.OpenDevTools();
```
Comment (Yuvix25) [inline Startup.cs:411]: Not needed - please remove.

File: Startup.cs [hunk 4]
```diff
@@ -148,7 +155,7 @@ private static async Task<BrowserWindow> CreateWindowAsync(BrowserWindowOptions
         return await Electron.WindowManager.CreateWindowAsync(options, loadUrl);
     }
 
-    private async Task CreateMainWindow()
+    private async Task CreateMainWindow(IWebHostEnvironment env)
```
Comment (Yuvix25) [inline Startup.cs:158]: Looks like you didn't end up using `env`, remove please.

File: Startup.cs (hunk 1 above)
Comment (rcbevans) [reply Startup.cs:37]: If we're attaching the debugger in program it's not needed here, I'll remove

Comment (rcbevans) [reply Startup.cs]: Removed

File: Startup.cs (hunk 2 above)
Comment (rcbevans) [reply Startup.cs]: Removed

File: Startup.cs (hunk 4 above)
Comment (rcbevans) [reply Startup.cs]: Removed

**Comment by rcbevans** [comment]
Addressed comments

**Pull request merged**